x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -101,15 +121,22 @@ function useServicesFetcher() {
   );
 
   return {
+    sortedAndFilteredServicesData,
+    sortedAndFilteredServicesStatus,
     mainStatisticsData,
     mainStatisticsStatus,
     comparisonData,
```

Comment:
Instead of separating `status` from `data` I think we should make it a convention to pass it around as a single object:

```suggestion
    sortedAndFilteredServicesResp,
    mainStatisticsResp,
    comparisonResp,
```

Reply:
I think that make sense. Should we append something like 'fetch', e.g. `sortedAndFilteredServicesFetch`?

Reply:
Yes, a `Fetch` suffix sgtm 👍